README.md

```diff
@@ -36,10 +36,10 @@ Reverse geocoding is the opposite: returning a list of places near a given point
 - Support multiple data sources (the defaults are [OpenStreetMap](http://openstreetmap.org/), [OpenAddresses](http://openaddresses.io), [Geonames](http://geonames.org), and [Who's on First](http://whosonfirst.mapzen.com/))
 - Flexible software architecture
 - Easy to contribute software patches and features to
-- Easy to set-up and configure your own instance
+- Easy to set up and configure your own instance
 - No external dependencies (such as postgres)
 - Reliable, configurable & fast import process
-- Work equally well for a small city and the entire planet
+- Works equally well for a small city and the entire planet
```

Comment:
I think "Work" is correct here because the section is a list of goals rather than a list of features. Does that sound right?